### ISSUE TITLE
Corpus integration?

### ISSUE THREAD

**Issue by redhae** [quran/quran.com-frontend#479]
just a feature suggestion; would be useful for someone learning arabic to have each word linked to the correponding corpus link (eg first word of quran.com/1:1 to http://corpus.quran.com/wordmorphology.jsp?location=(1:1:1)). Maybe linked via the translation tooltip?

**Comment by mmahalwy** [comment]
Salam! 

We do get this data in the api, just not using it unfortunately. We just need to figure out how to present it.

**Comment by ATouhou** [comment]
What about when adding the users portal, there could be an option to enable an "advanced interface "for people who want multiple tafaseer in arabic and english + grammar information shown ?

**Comment by mmahalwy** [comment]
Great idea! Let's add this as an option for the cookie first? Then with user auth we can go deeper?

**Comment by ATouhou** [comment]
Agreed. I think releasing tafaseer is a higher priority than the corpus display. However maybe they should be activated from the same place & API route? =)
